### ISSUE TITLE
[Bug] "command" parameter on sidecars appears to be ignored

### ISSUE THREAD

**Issue by FraserThompson** [aws/copilot-cli#3665]
As far as I can tell the `command` parameter on sidecars (documented [here](https://aws.github.io/copilot-cli/docs/developing/sidecars/#command)) doesn't actually override the command in the image.

To test this I added a sidecar with this simple definition:

```
  test:
    image: alpine
    command: "while [ ! ]; do echo 'test' && sleep 1; done"
```

I would expect it to hang around perpetually spitting "test" into stdout.

What happened instead was the sidecar exited immediately, indicating that my command wasn't being passed to the container.

Also, changing the command parameter and trying to deploy again netted a "stack has no changes" error, so it seems like the command parameter is just completely ignored.

**Comment by Lou1415926** [comment]
Hi @FraserThompson - thank you very much for letting us know! I was able to reproduce this bug - the command and entrypoint overrides only get rendered if `sidecar.port` is specified. We will fix it as soon as possible!

**Comment by efekarakus** [comment]
This fix is now out in v1.20 🚀 
Release notes: https://github.com/aws/copilot-cli/releases/tag/v1.20.0 !
Blog post: https://aws.github.io/copilot-cli/blogs/release-v120/
